w1thermsensor/core.py

```diff
@@ -101,14 +103,16 @@ def __init__(self, sensor_type=None, sensor_id=None):
             :raises KernelModuleLoadError: if the w1 therm kernel modules could not be loaded correctly
             :raises NoSensorFoundError: if the sensor with the given type and/or id does not exist or is not connected
         """
+        super(W1ThermSensor, self).__init__()
```

Comment:
From what I understand, it helps if you have a class derived from W1ThermSensor and something else. Basically, it's possible for Python to insert something above W1ThermSensor in the inheritance order in order to resolve the multiple inheritance. Something about Method Resolution Order or MRO.